pkg/landscaper/installations/helper.go

```diff
@@ -449,3 +450,57 @@ func ResolveConfigMapReference(ctx context.Context, kubeClient client.Client, co
 
 	return completeData, data, cm.Generation, nil
 }
+
+// HandleSubComponentPhaseChanges updates the phase of the given installation, if its phase doesn't match the combined phase of its subinstallations/executions anymore
+func HandleSubComponentPhaseChanges(ctx context.Context, kubeClient client.Client, inst *lsv1alpha1.Installation, getOperation func(context.Context, *lsv1alpha1.Installation) (*Operation, error), getExports func(context.Context, *Operation) ([]*dataobjects.DataObject, []*dataobjects.Target, error), logger logr.Logger) error {
```

Comment:
this looks unreadable on a non-widescreen.
Can you make it more readable like
```suggestion
func HandleSubComponentPhaseChanges(
  ctx context.Context,
  kubeClient client.Client,
  inst *lsv1alpha1.Installation,
  getOperation func(context.Context, *lsv1alpha1.Installation) (*Operation, error),
  getExports func(context.Context, *Operation) ([]*dataobjects.DataObject, []*dataobjects.Target, error), logger logr.Logger) error {
```